### ISSUE TITLE
Add Teacher working conditions data to database

### ISSUE THREAD

**Issue by jtf621** [codefordurham/school-navigator#366]
data located at: http://www.ncteachingconditions.org/results/report/153/62868 for teacher survey

https://durhamschoolnavigator.org/#/school/206/  Teacher working conditions

**Comment by nathanrogers18** [comment]
I'll take this.

**Comment by nathanrogers18** [comment]
tl;dr: We should have the raw survey data soon via the organization that conducted the survey.

I had some issues getting the excel files to download in a true xlsx format, rather than in html (they opened as Office Open XML spreadsheet).  

Since the xlsx files were a pain, I built a working BeautifulSoup scraper for one of the results pages, It was working, but then I realized they had newer data for the same schools in a slightly different format...

So I reached out to the New Teacher Center, the nonprofit that conducts these surveys.  They just got back to me and said they'd be happy to share the raw data files.  I think that will be much easier to work with than scraping the website data.  So for now, that's my plan.

**Comment by jtf621** [comment]
Sounds like a good plan.  Thanks.

**Comment by jtf621** [comment]
@nathanrogers18 any work on when the New Teacher Center will send the data?

**Comment by nathanrogers18** [comment]
NTC sent me a data sharing agreement today.  

The agreement specifies that we cannot publish any aggregate data on a level lower than school district.  

That was not my understanding when I first spoke with New Teacher Center, so I've reached out for clarification, and confirmation that we can aggregate results school level as published on ncteachingconditions.org.

**Comment by nathanrogers18** [comment]
Just chatted with NTC.  They confirmed that it is just their boilerplate contract language, and that the intent is for Code for Durham to publish aggregate data on a school level.  

The other significant item in the agreement is that we send them a copy of any data we plan to publish 7 days before publishing.  It also seems that this is boilerplate language, and I think a call to their contact would suffice.  

Is there a good email address I can send you the full agreement, and contact details for the representative at NTC?  My plan is to sign and return it today, and we should have our data soon after.

**Comment by jtf621** [comment]
Thanks. jfreeman@codeforamerica.org.  Let me review the contract.
